lib/geordi/gitpt.rb

```diff
@@ -24,6 +25,28 @@ def run(git_args)
     end
   end
 
+  def run_checkout
+    story = choose_story
+    return unless story
+
+    normalized_story_name = normalize_branch_name(story.name)
+
+    branch_list_string, _, branch_list_status = Open3.capture3('git', 'branch', '--format=%(refname:short)')
```

Comment:
Once you add tests, I think it is required to use some `Geordi::Util.run!` or a implementation, that does not execute this command in tests. See https://github.com/makandra/geordi/blob/c67008023b64dd88cbbd1bed0ddb3a7b64d97e58/lib/geordi/util.rb#L59-L61 for more details.

Reply:
Yes, please prefer the built-in `Util.run!`. It even takes a `fail_message:`.